src/hardware/vga_draw.cpp

```diff
@@ -970,10 +970,10 @@ static void VGA_DrawSingleLine(uint32_t /*blah*/)
 				write_unaligned_uint32_at(TempLine, i++, background_color);
 			}
 		}
-		RENDER_DrawLine(TempLine);
+		ReelMagic_RENDER_DrawLine(TempLine);
 	} else {
-		uint8_t * data=VGA_DrawLine( vga.draw.address, vga.draw.address_line );	
-		RENDER_DrawLine(data);
+		uint8_t * data=VGA_DrawLine( vga.draw.address, vga.draw.address_line );
+		ReelMagic_RENDER_DrawLine(data);
```

Comment:
The ReelMagic related cleanups/renames most definitely warrant their own commit.